ScanbotSDKexample/payform-scanner/src/main/java/io/scanbot/example/PayformResultActivity.java

```diff
@@ -0,0 +1,91 @@
+package io.scanbot.example;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.TextView;
+
+import net.doo.snap.payformscanner.entity.RecognizedField;
+
+import java.util.List;
+
+public class PayformResultActivity extends AppCompatActivity {
+    public static final String EXTRA_SENDER = "sender";
+    public static final String EXTRA_IBAN = "iban";
+    public static final String EXTRA_BIC = "bic";
+    public static final String EXTRA_AMOUNT = "amount";
+    public static final String EXTRA_REFERENCE_NUMBER = "reference_number";
+
+    private TextView sender;
+    private TextView iban;
+    private TextView bic;
+    private TextView amount;
+    private TextView referenceNumber;
+
+    public static Intent newIntent(Context context, String sender, String iban, String bic, String amount, String referenceNumber) {
+        Intent intent = new Intent(context, PayformResultActivity.class);
+        intent.putExtra(EXTRA_SENDER, sender);
+        intent.putExtra(EXTRA_IBAN, iban);
+        intent.putExtra(EXTRA_BIC, bic);
+        intent.putExtra(EXTRA_AMOUNT, amount);
+        intent.putExtra(EXTRA_REFERENCE_NUMBER, referenceNumber);
+        return intent;
+    }
+
+    public static Intent newIntent(Context context, List<RecognizedField> fields) {
+        Intent intent = new Intent(context, PayformResultActivity.class);
+        for (RecognizedField field : fields) {
+            switch (field.getTokenType()) {
+                case RECEIVER:
+                    intent.putExtra(EXTRA_SENDER, field.getValue());
+                    break;
+                case IBAN:
+                    intent.putExtra(EXTRA_IBAN, field.getValue());
+                    break;
+                case AMOUNT:
+                    intent.putExtra(EXTRA_AMOUNT, field.getValue());
+                    break;
+                case BIC:
+                    intent.putExtra(EXTRA_BIC, field.getValue());
+                    break;
+                case REFERENCE_NUMBER:
```

Comment:
Please also add `REFERENCE_NUMBER2` and show it in the UI. This is the second line of the reference number.